CLI/test.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
```

Comment:
Should this be a Django TestCase or do `import unittest`?

Reply:
Shouldn't have added this test file. I was testing a few things but still not commit material. I will remove these.